### ISSUE TITLE
RFC: alt tab ordering should be configurable

### ISSUE THREAD

**Issue by ondreian** [elanthia-online/illthorn#71]
I'm not sure how to do this just yet in a scale-able, intuitive manner, but we should make it so that there is a completely configurable `alt + <Integer>` so it's not 100% dependent on login order.

Drag-n-drop might be the most intuitive manner for configuring these values, I'm not sure yet.

Anyone have thoughts on how to handle this?

**Comment by chriscoyier** [comment]
I still very much like the idea of drag-and-drop for the panel system, which then saves that sort order in the Settings data store. So with that in mind, perhaps whatever system is devised should handle a drag-and-drop + sort in a generic re-usable way that could be used here too. 

What maybe complicates it is that not all accounts are always logged in. So if you have a character you usually play on ALT-2 but ALT-1 isn't logged in, does that ALT-2 character become ALT-1 or stay a 2?

**Comment by ondreian** [comment]
> What maybe complicates it is that not all accounts are always logged in. So if you have a character you usually play on ALT-2 but ALT-1 isn't logged in, does that ALT-2 character become ALT-1 or stay a 2?

I've been thinking a lot about this, and I think they should "stay" at whatever position they are assigned, but we need to gracefully handle conflicts.

Scenarios:
```
alt + 1 = Jack (Account1)
alt + 2 = Alfred (Account2)
alt +3 = Jill (Account3)
```
If someone assigns `Eric (Account2)` to `alt +1` there should be a conflict, and the first session "seen" should win out and the conflicting session should get assigned the next available position.

This allows for any team to be setup to always have the same "order" and macros, even when maybe team members aren't all logged in.
